client/data/sites/use-site-excerpts-query.ts

```diff
@@ -69,7 +69,7 @@ function computeFields( allSites: SiteExcerptNetworkData[] ) {
 
 		return {
 			...data,
-			name: trimmedName.length > 0 ? trimmedName : slug,
+			title: trimmedName.length > 0 ? trimmedName : slug,
```

Comment:
This is so we can have consistency between the Redux store and the `react-query` Realm. The Redux store computes the `name` and `slug` as `title` instead of overwriting the `name`, and we should do the same.

Doing that allows us to re-use the sorting mechanism between the data stores as the pieces of data, even the computed ones, now match.